test/parallel/test-fs-promises-file-handle-close-errors.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+// Flags: --expose-internals
+
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+
+// The following tests validate aggregate errors are thrown correctly
+// when both an operation and close throw.
+
+const fs = require('fs');
+const path = require('path');
+const {
+  readFile,
+  writeFile,
+  truncate,
+  lchmod,
+} = fs.promises;
+const {
+  FileHandle,
+} = require('internal/fs/promises');
+
+const assert = require('assert');
+const originalFd = Object.getOwnPropertyDescriptor(FileHandle.prototype, 'fd');
+
+let count = 0;
+async function createFile() {
+  const filePath = path.join(tmpdir.path, `close_errors_${++count}.txt`);
+  await writeFile(filePath, 'content');
+  return filePath;
+}
+
+async function checkAggregateError(op) {
```

Comment:
Maybe the name should be changed here to reflect the fact that it won't be an `AggregateError`.